azure-pipelines.yml

```diff
@@ -38,15 +49,16 @@ jobs:
   - task: NodeTool@0
     inputs:
       versionSpec: '14.x'
-  - task: PowerShell@2
-    displayName: 'Install .NET 6'
-    inputs:
-      targetType: 'inline'
-      script: |
-        Invoke-WebRequest 'https://dot.net/v1/dotnet-install.ps1' -OutFile 'dotnet-install.ps1'
-        # Official release versions can be found at: https://dotnet.microsoft.com/download/dotnet/6.0
-        # Newer versions can be found at: https://github.com/dotnet/installer#installers-and-binaries
-        ./dotnet-install.ps1 -InstallDir 'C:\Program Files\dotnet' -Verbose -Version 6.0.100
+  - pwsh: |
+      Import-Module "$PSScriptRoot/helper.psm1" -Force
+      Install-Dotnet
+    displayName: 'Install .NET 6.0 and 3.1'
+  - pwsh: |
+      Import-Module "$PSScriptRoot/helper.psm1" -Force
+      Install-SBOMUtil $env:SBOMUtilSASUrl
```

Comment:
Please include the parameter name. E.g., `Install-SBOMUtil -SBOMUtilSASUrl $env:SBOMUtilSASUrl`